class/User.js

```diff
@@ -326,11 +331,7 @@ export class User {
    * @returns {Promise<Array>}
    */
   async getTxs() {
-    let addr = await this.getAddress();
-    if (!addr) {
-      await this.generateAddress();
-      addr = await this.getAddress();
-    }
+    const addr = await this.getOrGenerateAddress();
     if (!addr) throw new Error('cannot get transactions: no onchain address assigned to user');
```

Comment:
this string is now inside  `getOrGenerateAddress()` so it should be safe to remove it from here